source/Outfit.cpp

```diff
@@ -244,6 +263,29 @@ void Outfit::Load(const DataNode &node)
 
 
 
+void Outfit::FinishLoading()
+{
+	const Outfit *parent = GameData::Outfits().Get(parentName);
+		
+	// Copy data over from the parent outfit.
+	if(!hasCategory)
+		category = parent->Category();
+	if(!hasThumbnail)
+		thumbnail = parent->Thumbnail();
+	if(!hasCost)
+		cost = parent->Cost();
+	if(!hasMass)
+		mass = parent->Mass();
```

Comment:
These booleans are redundant with other methods you could use to check if these have been set. I've talked about looking to how ship variants are handled and emulating that here, and in that case there aren't any additional booleans checking if various attributes have been set or not.
https://github.com/endless-sky/endless-sky/blob/c9a79a3080fc7a52d1123385f5fdafc451fc73f3/source/Ship.cpp#L490-L500
```suggestion
	if(category.empty())
		category = parent->Category();
	if(!thumbnail)
		thumbnail = parent->Thumbnail();
	if(!cost)
		cost = parent->Cost();
	if(!mass)
		mass = parent->Mass();
```

Reply:
They do allow setting the respective items to values that implicitly convert to `false`